clients/clearly-defined/src/funTest/kotlin/ClearlyDefinedServiceFunTest.kt

```diff
@@ -19,12 +19,17 @@
 
 package org.ossreviewtoolkit.clients.clearlydefined
 
+import com.github.tomakehurst.wiremock.WireMockServer
```

Comment:
This file is a *FunTest*, i.e. for functional tests. So far we only use mocks in unit tests, and I'd prefer to keep that split. So please either keep the test to be a functional one without mocking, or move tests that use mocking to a new `clients/clearly-defined/src/test/kotlin/ClearlyDefinedServiceTest.kt`.

Reply:
The test case was rewritten to just test the serialization. The dependency to the mock server was removed.